_data/simple-icons.json

```diff
@@ -1238,6 +1238,16 @@
             "hex": "FFB80B",
             "source": "https://partnerships.betfair.com/"
         },
+        {
+            "title": "Bevy",
+            "hex": "232326",
+            "source": "https://bevyengine.org",
+            "aliases": {
+                "aka": [
+                    "Bevy Engine"
+                ]
+            }
```

Comment:
Do we need the alias? The only places I can see where they refer to themselves as "Bevy Engine" are in their [website](https://bevyengine.org/)'s `<title>` and the name of [their GH org](https://github.com/bevyengine/).
```suggestion
```